test-cases/tests/documentation/template.md

```diff
@@ -0,0 +1,20 @@
+<Test Case Number> - <Jira Title>
+
+* Sprint # 
+
+## Description
+
+<Summary>
+
+* Jira ref link:
```

Comment:
This should be a link to the module or modules on our staging server. We can also add the JIRA link if you think it will be helpful to have that relationship preserved.